docs/source/usage.rst

```diff
@@ -86,108 +86,109 @@ Here is a commented example:
-          type: virtualbox
-          options:
-            memory: 512
-            cpus: 2
-
-    # defaults passed to ansible-playbook
-    ansible:
-      user: vagrant
-      connection: ssh
-      timeout: 30
-      playbook: playbook.yml
-      sudo: True
-      sudo_user: False
-      ask_sudo_pass: False
-      ask_vault_pass: False
-      vault_password_file: False
-      limit: all
-      verbose: False
-      diff: True
-      tags: False
-      host_key_checking: False
-      raw_ssh_args:
-        - -o UserKnownHostsFile=/dev/null
-        - -o IdentitiesOnly=yes
-        - -o ControlMaster=auto
-        - -o ControlPersist=60s
+	molecule:
+	  # file to look for in CWD when using ansible-playbook directly
+	  molecule_file: molecule.yml
+
+	  # directories to ignore when doing trailing whitespace checks on files during verify command
+	  ignore_paths:
+		- .git
+		- .vagrant
+		- .molecule
+
+	  # directory to look for serverspec tests
+	  serverspec_dir: spec
+
+	  # directory to look for testinfra tests
+	  testinfra_dir: tests
+
+	  # directory in CWD to place all temp files, etc.
+	  molecule_dir: .molecule
+
+	  # where temporary state will be stored (lives under molecule_dir)
+	  state_file: state
+
+	  # name of inventory file generated by molecule (lives under molecule_dir)
+	  inventory_file: ansible_inventory
+
+	  # name of ansible config file generated by molecule (lives under molecule_dir)
+	  config_file: ansible.cfg
+
+	  # name of temporary vagrantfile created during runs (lives under molecule_dir)
+	  vagrantfile_file: vagrantfile
+	  rakefile_file: rakefile
+
+	  # template files to load when creating corresponding temporary files
+	  # this would be a good place to specify your own ansible.cfg template, for example
+	  vagrantfile_template: vagrantfile.j2
+	  ansible_config_template: ansible.cfg.j2
+	  rakefile_template: rakefile.j2
+
+	  # default provider to use when no --provider flag is specified
+	  # comment this out to default to the first in the provider list
+	  # default_provider: virtualbox
+
+	  # default platform to use when no --platform flag is specified
+	  # comment this out to default to the first in the platform list
+	  # default_platform: rhel-7
+
+	  # ssh arguments passed to molecule login command
+	  raw_ssh_args:
+		- -o StrictHostKeyChecking=no
+		- -o UserKnownHostsFile=/dev/null
+
+	  test:
+		# sequence of commands to run when performing `molecule test`
+		sequence:
+		  - destroy
+		  - create
+		  - converge
+		  - idempotence
+		  - verify
+
+	  init:
+		# default platform to populate when doing `molecule init`
+		platform:
+		  name: trusty64
+		  box: trusty64
+		  box_url: https://vagrantcloud.com/ubuntu/boxes/trusty64/versions/14.04/providers/virtualbox.box
+		# templates to use when creating files during `molecule init`
+		templates:
+		  molecule: molecule.yml.j2
+		  playbook: playbook.yml.j2
+		  spec_helper: spec_helper.rb.j2
+		  default_spec: default_spec.rb.j2
```

Comment:
This looks wrong from master.

`spec_helper` and `default_spec` are removed in favor of `test_default`.